bom/application/pom.xml

```diff
@@ -164,7 +164,7 @@
         <kotlin.coroutine.version>1.8.0</kotlin.coroutine.version>
         <azure.toolkit-lib.version>0.27.0</azure.toolkit-lib.version>
         <kotlin-serialization.version>1.6.2</kotlin-serialization.version>
-        <dekorate.version>4.1.2</dekorate.version> <!-- Please check with Java Operator SDK team before updating -->
+        <dekorate.version>4.1.3</dekorate.version> <!-- Please check with Java Operator SDK team before updating -->
```

Comment:
@gsmet, should this change be on separated pull request?

Reply:
@metacosm I assume the version bump is fine with you?

Reply:
I actually don't know and didn't get a chance to check… That version of dekorate uses an older version than the one used in the Kubernetes client and I have no idea about compatibility. I would have preferred this upgrade to be isolated from this PR if at all possible and be consulted before merging it, as is mentioned in the POM file.

Reply:
Fair enough. 

When you get please chance, please check the version and see if everything works as expected.

Thanks!